test/bin/compiler-beta.t/run.t

```diff
@@ -0,0 +1,132 @@
+We have a simple project that only depends on OCaml
+
+  $ cat test.opam
```

Comment:
I think printing the opam files entirely is not great, because the files are in Git anyway so the user can take a look at them. Rather I would suggest using the opam tool to print the interesting bit of the file, so the file can be modified if something unrelated needs to change without disrupting all the tests.

Reply:
This file is short but it also includes a bunch of boilerplate, like the way the repos are set up which has nothing to do with the test and were it to change the test would need to be adapted for reasons that have nothing to do with the test itself.

Reply:
I'll remove them but I don't see any reason why they would need to change since they are in fact very specific to this test and it's actually quite important to know what metadata will be used by the solver.

That's one of the reason why you can tell that it will always pick `4.13.1` below for instance.